entries/r/racq.com.au.json

```
@@ -0,0 +1,15 @@
+{
+  "RACQ": {
+    "domain": "racq.com.au",
+    "contact": {
+      "twitter": "RACQOfficial",
+      "facebook": "racqofficial"
+    },
+    "categories": [
+      "insurance"
```

```suggestion
	  "banking",
      "insurance"
```

Looks like they offer [banking services](https://www.racq.com.au/banking) as well

---

That is true, and I was going to make a separate listing for it as it seems to be a different authentication system. There seems to be 2FA available for the banking login, the domain racqbank.com.au doesn't meant the rank criteria though.

---

Got it.